unittests/builtinhttp/builtinhttp.cpp

```diff
@@ -426,6 +426,38 @@ private Q_SLOTS:
         }
     }
 
+    // Test that that if the setUseWSAddressing is set int the client  then the SOAP action
+    // is really placed to the SOAP envelope's header and not embedded to the  Content-Type HTTP header
+    void testUseWSAddressingCall()
+    {
+        HttpServerThread server(countryResponse(), HttpServerThread::Public);
+        server.setClientUseWSAddressing();
+
+        qDebug() << "server ready, proceeding" << server.endPoint();
+        KDSoapClientInterface client(server.endPoint(), countryMessageNamespace());
+        client.setSendHTTPActionParameter();
```

Comment:
That's the default, so this line is a no-op.

What is actually missing is a test for setSendHTTPActionPameter(false), isn't it?

Reply:
Yes, actually I submitted to review before fixing the unit tests. Now I think it should be ok including the tests.